### PR TITLE
Update AssemblyScript code snippets in snake tutorial

### PR DESCRIPTION
**Pull request by MaxGraey**
This updates following recent changes in https://github.com/christopher-kleine/wasm-4-tutorial-games
Also better format and fix couple typos.

### CODE REVIEW

**Comment by MaxGraey** [comment]
cc @christopher-kleine

File: site/docs/tutorials/snake/placing-the-fruit.md [hunk 1]
```diff
@@ -201,11 +201,11 @@ To get it into a an existing file, use the `>>` operator. Like this:
...
-var snake = new Snake()
-var frameCount = 0
-var prevState : u8
-var fruit = new Point(rnd(20), rnd(20))
-const fruitSprite = memory.data<u8>([ 0x00,0xa0,0x02,0x00,0x0e,0xf0,0x36,0x5c,0xd6,0x57,0xd5,0x57,0x35,0x5c,0x0f,0xf0 ]);
+const snake = new Snake()
+const fruit = new Point(rnd(20), rnd(20))
+let frameCount = 0
+let prevState: u8
+const fruitSprite = memory.data<u8>([ 0x00,0xa0,0x02,0x00,0x0e,0xf0,0x36,0x5c,0xd6,0x57,0xd5,0x57,0x35,0x5c,0x0f,0xf0 ])
```
[the remark this answers is not in the document]
Comment (MaxGraey) [reply site/docs/tutorials/snake/placing-the-fruit.md]: thanks!

**Comment by aduros** [comment]
I think we should keep the indentation at 4 spaces for consistency with the rest of the documentation.

**Comment by MaxGraey** [comment]
@aduros Done!

**Comment by aduros** [comment]
Thanks!

**Pull request merged**